setup/controllers/language.php

```diff
@@ -35,7 +35,9 @@
     $settings = $install->request->getConfig();
     $settings = array_merge($settings, $_POST);
     $install->settings->store($settings);
-    $this->proceed('welcome');
+
+    $nextAction = (MODX_SETUP_KEY === '@traditional@') ? 'options' : 'welcome';
```

Comment:
Excess brackets.

Reply:
@mishantrop Thanks for taking time to review my code. I think the brackets make the code easier to read, don't you agree?

Reply:
Without [codings stardarts](https://docs.modx.com/current/en/contribute/code/coding-standards) for this, brackets is a matter of taste. Without brackets, the code looks more concise; it takes less time to read.
At this moment, that's at your discretion.